src/BookNavigator/volumes/volumes-provider.js

```diff
@@ -24,40 +24,52 @@ export default class VolumesProvider {
     this.id = 'volumes';
     this.label = `Viewable files (${this.volumeCount})`;
     this.icon = html`${volumesIcon}`;
-    this.actionButton = this.headerIcon;
-    this.sortVolumes(true);
+
+    this.sortOrderBy = "";
+    this.sortVolumes("initial");
+  }
+
+  get sortInitialIcon() {
+    return html`<button class="sort-by neutral-icon" aria-label="Sort volumes in initial order" @click=${() => this.sortVolumes('asc')}>${sortNeutralIcon}</button>`;
   }
 
   get sortAscendingIcon() {
-    return html`<button class="sort-asc icon" aria-label="Sort volumes in ascending order" @click=${() => this.sortVolumes()}>${sortAscendingIcon}</button>`;
+    return html`<button class="sort-by asc-icon" aria-label="Sort volumes in ascending order" @click=${() => this.sortVolumes('desc')}>${sortAscendingIcon}</button>`;
   }
 
   get sortDescendingIcon() {
-    return html`<button class="sort-desc icon" aria-label="Sort volumes in descending order" @click=${() => this.sortVolumes()}>${sortDescendingIcon}</button>`;
+    return html`<button class="sort-by desc-icon" aria-label="Sort volumes in descending order" @click=${() => this.sortVolumes('initial')}>${sortDescendingIcon}</button>`;
   }
 
   get headerIcon() {
-    return this.isSortAscending ? this.sortAscendingIcon : this.sortDescendingIcon;
+    if (this.sortOrderBy === 'initial') return this.sortInitialIcon;
+    else if (this.sortOrderBy === 'asc') return this.sortAscendingIcon;
+    else return this.sortDescendingIcon;
   }
 
-  sortVolumes(initialSort = false) {
-    this.isSortAscending = !this.isSortAscending;
-    const volumesOrderBy = this.isSortAscending ? 'asc' : 'desc';
+  /**
+   * @param {string} sortType (initial, asc, desc)
+   */
+  sortVolumes(sortType) {
+    let sortedFiles = [];
 
-    const sortedFiles = this.viewableFiles.sort((a, b) => {
-      if (this.isSortAscending) return a.title.localeCompare(b.title);
+    sortedFiles = this.viewableFiles.sort((a, b) => {
+      if (sortType === 'initial') return a.orig_sort - b.orig_sort;
+      else if (sortType === 'asc') return a.title.localeCompare(b.title);
```

Comment:
Hmm, not sure why codecov is not realizing this line was run :/ It must've been by your test --- update: I think the test isn't running!